exporter/tinybirdexporter/README.md

```diff
@@ -11,31 +11,55 @@
 [development]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/component-stability.md#development
 <!-- end autogenerated section -->
 
-This is the initial version of the Tinybird Exporter that sends data to Tinybird using the Events API. Still under active development
+## Description
+
+This exporter sends logs, metrics, and traces to [Tinybird](https://www.tinybird.co/) via the [Events API](https://www.tinybird.co/docs/api-reference/events-api).
+
+Tinybird is a real-time analytics platform for ingesting, transforming, and serving data with low latency and high throughput. Telemetry data sent to Tinybird can be queried and analyzed in real time using SQL.``
+
+
+## Quick Start
+
+Looking for a quick start? Check out our [Open Telemetry Template](https://github.com/tinybirdco/tinybird-otel-template), which provides a ready-to-use example for configuring the Tinybird OpenTelemetry Exporter.
 
 ## Configuration
-- `endpoint`: Tinybird API endpoint
-- `token`: Tinybird API token
-- `metrics`:
-  - `gauge`
-    - `datasource` (default `gauge`): Name of the gauge metrics datasource
-  - `sum`
-    - `datasource` (default `sum`): Name of the sum metrics datasource
-  - `histogram`
-    - `datasource` (default `histogram`): Name of the histogram metrics datasource
-  - `exponential_histogram`
-    - `datasource` (default `exponential_histogram`): Name of the exponential histogram metrics datasource
-- `traces`
-  - `datasource` (default `traces`): Name of the traces datasource
-- `logs`
-  - `datasource` (default `logs`): Name of the logs datasource
-
-Example:
+
+### Required settings
+
+- `endpoint` (no default): The Tinybird API endpoint URL (e.g., `https://api.us-east.aws.tinybird.co`)
+- `token` (no default): Your Tinybird API token used for authentication
+
+### Optional settings
+
+- `metrics`: Configuration for different metric types
+  - `gauge.datasource` (default: `gauge`): Name of the gauge metrics data source
+  - `sum.datasource` (default: `sum`): Name of the sum metrics data source  
+  - `histogram.datasource` (default: `histogram`): Name of the histogram metrics data source
+  - `exponential_histogram.datasource` (default: `exponential_histogram`): Name of the exponential histogram metrics data source
+- `traces.datasource` (default: `traces`): Name of the traces data source
```

Comment:
You should use `::` instead of `.` for namespacing, `.` will not work
```suggestion
  - `gauge::datasource` (default: `gauge`): Name of the gauge metrics data source
  - `sum::datasource` (default: `sum`): Name of the sum metrics data source  
  - `histogram::datasource` (default: `histogram`): Name of the histogram metrics data source
  - `exponential_histogram::datasource` (default: `exponential_histogram`): Name of the exponential histogram metrics data source
- `traces::datasource` (default: `traces`): Name of the traces data source
```